magento/pwa-studio
# PWA-922: Updated splash image Venia Closed

## Description

Update "Venia Closed" splash image

## Related Issue







Closes [PWA-922](https://jira.corp.magento.com/browse/PWA-922).

## Acceptance

When the Venia storefront is visited with JavaScript disabled, the user sees the updated Venia image instead of the old logo

### Verification Stakeholders



### Specification




## Verification Steps

- Open Venia
- Turn off JS in your browser
- Verify that the blue "V" with the crossed out JS image appears instead of the old Venia with a flower logo and closed sign

#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
